PMM-13915-update-golang-libs (FB)

BupycHuk opened the pull request:
Custom branches: 
postgres_exporter
azure_metrics_exporter

BupycHuk committed:
Create feature build: PMM-13915-update-golang-libs

JNKPercona commented:
Server docker: perconalab/pmm-server-fb:PR-3893-0d92db7
Client docker: perconalab/pmm-client-fb:PR-3893-0d92db7
Watchtower docker: perconalab/pmm-watchtower-fb:PR-3893-0d92db7
Client tarball: https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm-client/pmm-client-PR-3893-0d92db7.tar.gz
Staging instance: https://pmm.cd.percona.com/job/pmm3-aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3893-0d92db7&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm-client/pmm-client-PR-3893-0d92db7.tar.gz

JNKPercona commented:
API tests have failed: https://pmm.cd.percona.com/job/pmm3-api-tests/2417/

BupycHuk committed:
Merge branch 'v3' into PMM-13915-update-golang-libs

BupycHuk committed:
PMM-13915 Add proxysql_exporter to CI configuration

Included proxysql_exporter with the specified branch and repository URL. This aligns with updating Go libraries under the PMM-13915 task.

BupycHuk committed:
Create feature build: PMM-13915-update-golang-libs
